src/LaunchDarkly.Client/packages.config

```diff
@@ -5,6 +5,6 @@
   <package id="Microsoft.Bcl" version="1.1.10" targetFramework="net45" />
   <package id="Microsoft.Bcl.Build" version="1.0.21" targetFramework="net45" />
   <package id="Microsoft.Net.Http" version="2.2.29" targetFramework="net45" />
-  <package id="Newtonsoft.Json" version="8.0.3" targetFramework="net45" />
+  <package id="Newtonsoft.Json" version="8.0.3" targetFramework="net451" />
```

Comment:
As mentioned above this is what we had in the prior versions of the .NET client when there were no reports of the Newtonsoft dependency failing to be brought in correctly.